extensions/eclipse-che-theia-plugin-ext/src/plugin/che-api.ts

```diff
@@ -9,16 +9,10 @@
  **********************************************************************/
 
 import * as che from '@eclipse-che/plugin';
+import { che as cheApi } from '@eclipse-che/api';
 import { RPCProtocol } from '@theia/plugin-ext/lib/api/rpc-protocol';
 import { Plugin } from '@theia/plugin-ext/lib/api/plugin-api';
-import {
-    Workspace,
-    WorkspaceConfig,
-    ResourceCreateQueryParams,
-    WorkspaceSettings,
-    Variable,
-    Disposable
-} from '@eclipse-che/plugin';
+import { Variable, Disposable, KeyValue } from '@eclipse-che/plugin';
```

Comment:
we have 
```
import * as che from '@eclipse-che/plugin';
```
and
```
import { Variable, Disposable, KeyValue } from '@eclipse-che/plugin';
```

it looks like the second line of imports should not be used

Reply:
Done